questions/services.py

```diff
@@ -237,16 +237,26 @@ def create_conditional(
     condition = Question.objects.get(pk=condition_id)
     condition_child = Question.objects.get(pk=condition_child_id)
 
+    question_yes = clone_question(
+        condition_child, title=f"{condition.title} (Yes) → {condition_child.title}"
+    )
+    question_yes.scheduled_close_time = min(
+        condition.scheduled_close_time, condition_child.scheduled_close_time
+    )
```

Comment:
A small nit: I think it makes sense to modify `clone_question` to accept extra params in `kwargs` instead of cloning and modifying consequently